pkg/analyzer/pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: analyzer
-version: 0.29.6
+version: 0.29.6+1
```

Comment:
I don't understand the "+1". I was planning on making it "0.29.7". Can you explain the difference?